src/qmflows/utils.py

```diff
@@ -77,6 +79,62 @@ def wrapper(settings, key, value, mol, **kwargs):
     return wrapper
 
 
+def file_to_context(file: Union[PathLike, IO], allow_iterator: bool = True,
```

Comment:
This seems very useful

Reply:
Yeah, I felt I was reusing quite a bit of code so is just put it in ``qmflows.utils``.